### ISSUE TITLE
voice controls & accessbility mode

### ISSUE THREAD

**Issue by elevchyt** [elevchyt/openwar3#132]
We want to make the game as accessible as possible.
For this, we want to enable a "voice command accessbility mode".
Basically, players should be able to use their mics to interact with the game and be able to play as well as possible.
Labeling units and buildings (at least player owned ones) would be good for more specific actions.
